x-pack/plugins/license_management/public/plugin.ts

```diff
@@ -27,13 +30,15 @@ interface PluginsDependenciesStart {
 
 export interface LicenseManagementUIPluginSetup {
   enabled: boolean;
+  locator: undefined | LicenseManagementLocator;
```

Comment:
in which cases would the locator be undefined?

Reply:
For the case that the License Management plugin is hidden from the UI with `ui.enabled: false`. The plugin is still enabled, but we don't register it in the Kibana navigation and don't initialize the locator.